pkg/query-service/model/response.go

```diff
@@ -430,16 +430,16 @@ type GetFieldsResponse struct {
 type GetLogsResponse struct {
 	Timestamp          uint64             `json:"timestamp" ch:"timestamp"`
 	ID                 string             `json:"id" ch:"id"`
-	TraceID            string             `json:"traceId" ch:"trace_id"`
-	SpanID             string             `json:"spanId" ch:"span_id"`
-	TraceFlags         uint32             `json:"traceFlags" ch:"trace_flags"`
-	SeverityText       string             `json:"severityText" ch:"severity_text"`
-	SeverityNumber     uint8              `json:"severityNumber" ch:"severity_number"`
+	TraceID            string             `json:"trace_id" ch:"trace_id"`
+	SpanID             string             `json:"span_id" ch:"span_id"`
+	TraceFlags         uint32             `json:"trace_flags" ch:"trace_flags"`
+	SeverityText       string             `json:"severity_text" ch:"severity_text"`
+	SeverityNumber     uint8              `json:"severity_number" ch:"severity_number"`
```

Comment:
@nityanandagohain changing of keys in json serdes should be a breaking change. I wonder why no relevant other changes have to be done?

Reply:
So the issue that this PR fixes was due to the fact that these were camel case so we updated them.
They are mapped to static field constants, so they are not a breaking change as we tested queries with the static fields.

Tested it with @pranshuchittora